tap_github/client.py

```diff
@@ -105,6 +105,10 @@ def raise_for_error(resp, source, stream, client, should_skip_404):
     except JSONDecodeError:
         response_json = {}
 
+    if stream == "commits" and response_json.get("message") == "Git Repository is empty.":
+        LOGGER.warning("Encountered an empty git repository")
```

Comment:
Is there value in this log line? As-is, it seems unactionable, and just potentially confusing in the logs if the tap can continue successfully.

Reply:
Maybe the warning is a little dramatic. I think having the log line is valuable to know that this situation occurs.

The alternative is seeing that some configured repo didn't produce any commit records, I think

Reply:
I think logger.info would be useful because it can help support point to lack of discrepancy